WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewModel.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import Yosemite
+
+/// Provides view data for Add Attributes, and handles init/UI/navigation actions needed.
+///
+final class AddAttributeOptionsViewModel {
```

Comment:
would be nice having unit tests for the view model! (feel free to leave them for later)

Reply:
Yes, they will be added later, when we will have a real logic around it :) for the moment all the tests would be mock.